lib/events.js

```diff
@@ -20,6 +20,17 @@ EventEmitter.prototype._maxListeners = undefined;
 // added to it. This is a useful default which helps finding memory leaks.
 EventEmitter.defaultMaxListeners = 10;
 
+function ensureEvents(emitter) {
```

Comment:
`ensureXxx` function should not have a side-effect (no modified state).
I feel `initializeXxx` is better or remove side-effects like:

```
function ensureEvents(emitter) {
  if (!emitter._events ||
      emitter._events === Object.getPrototypeOf(emitter)._events) {
    return false;
  }
  return true;
}
```